lte/gateway/c/core/oai/common/common_defs.h

```diff
@@ -120,6 +120,35 @@ typedef enum {
   S8_SUBSCRIBER    = 2,  // will use federated HSS + s8_task
 } mode_map;
 
+typedef enum {
+  value_not_used    = 000000000,
+  Multiples_1kbps   = 0b00000001,
```

Comment:
these enum values should be on uppercase

Reply:
+1

Reply:
Sure, I'll update it.

Reply:
Comment addressed.